src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    private Bulldozer bulldozer1 = new Bulldozer(false, true, "CatLGP", "Yellow");
+    private Bulldozer bulldozer2 = new Bulldozer(false, false, "LiebherrPR734", "Green");
+    private Bulldozer bulldozer3 = new Bulldozer(true, true, "CatC15", "Red");
```

Comment:
make these fields local or maybe make them static, because every creation of Bulldozer producer will create new fields.